docker/docker-compose.yml

```diff
@@ -5,3 +5,8 @@ services:
     expose:
       - 8080
     restart: always
+  job-service:
+    build: ../job-service/server
+    expose:
+      - 8080
+    restart: always
```

Comment:
Could you please also add `depends_on` to `api_gateway` in `docker-compose.override.yml` file, we forgot to do that for file service:
```
version: '3'
services:
  api-gateway:
    build:
      context: ./api-gateway
      dockerfile: Dockerfile
      args:
        profile: ${PROFILE}
    ports:
      - 80:80
    depends_on:
      - local-storage-service
      - job-service
```